packages/web-twig/src/Resources/components/Collapse/stories/CollapseHideTrigger.twig

```diff
@@ -0,0 +1,31 @@
+<section class="docs-Section">
+
+    <h2 class="docs-Heading">Collapse with hide trigger when expanded </h2>
+
+    <!--
+        Collapse more
+    -->
+    <p>
+        Quis orci molestie, condimentum metus a lorem lorem fringilla sem lobortis libero rhoncus nullam, purus sapien aenean sed id tortor. Molestie quam, lectus quis ac nec
+        suspendisse consectetur finibus a accumsan, bibendum dolor sit amet congue rhoncus erat. Semper turpis, varius iaculis venenatis ut congue sollicitudin nulla,
+        convallis nibh maximus erat id aliquam. Nec interdum, auctor elit vitae pulvinar fusce gravida curabitur odio leo, nunc a mauris auctor. Vel integer, placerat pulvinar
+        nisi nunc donec felis erat quis, phasellus ac nec phasellus ac nibh. Scelerisque  fringilla, viverra nisl vel purus sed porttitor porttitor et iaculis commodo, vehicula urna purus ultrices.
+        <ButtonLink
+            href="javascript:void(0)"
+            data-toggle="collapse"
+            data-target="CollapseExampleIII"
+            data-more
+        >
+            ...more
+        </ButtonLink>
+        <Collapse elementType="span"
+            id="CollapseExampleIII"
```

Comment:
can you please improve line wrapping here? either keep everything on one line or each prop on separate line. Thanks